packages/superset-ui-core/src/color/SequentialScheme.ts

```diff
@@ -52,6 +52,7 @@ export default class SequentialScheme extends ColorScheme {
     const piecewiseScale: (t: number) => string = piecewise(interpolateHcl, this.colors);
     const adjustExtent = scaleLinear().range(extent).clamp(true);
 
+    // @ts-ignore
```

Comment:
Was seeing typescript errors here, but wasn't feeling inclined to tackle it in this PR. Suggestions welcome :)